src/modules/Tab/Tab.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import uuid from "uuid";
+import {
+  StandardProps,
+  WithStylesProps,
+  ThemeStyleSheetFactory
+} from "../../types";
+import { withStyles } from "../../base";
+
+export interface IProps extends StandardProps<"li"> {
+  disabled?: boolean;
+  id: string;
+  selectedTab?: string;
+  onTabClick?: (id: string) => void;
+}
+
+const styleSheet: ThemeStyleSheetFactory = theme => ({
+  tab: {
+    cursor: "pointer",
+    listStyle: "none",
+    marginRight: theme.spaceUnit.md,
+    maxWidth: "50px",
+    "@selectors": {
+      "&.selected": {
+        borderBottom: "1px solid black"
+      },
+      "&.disabled": {
+        cursor: "default"
+      }
+    }
+  }
+});
+
+const Tab: React.FC<IProps & WithStylesProps> = ({
+  children,
+  className,
+  cx,
+  disabled,
+  id = uuid(),
+  selectedTab,
+  styles,
+  onTabClick = () => {}
+}) => {
+  const handleClick = () => !disabled && onTabClick(id);
+  const stylesArray = cx(
+    styles.tab,
+    selectedTab === id && "selected",
+    disabled && "disabled",
+    className
+  );
+
+  return (
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
```

Comment:
is this rule hard to comply with?

Reply:
We can use the `role` attribute. Either role="button" or role="presentation"

Reply:
TBH this was just laziness, we have this issue in other places so I didn't bother trying to fix it.
I can look into your suggestion 👍

Reply:
I checked it out. I think having `role="button"` makes sense, but we get another lint error saying `Non-interactive elements should not be assigned interactive roles.` I'd rather ignore this error and leave it with `role="button"` (should we even turn this rule off??)